src/lib/containers/SynapseNavDrawer.tsx

```diff
@@ -226,71 +238,81 @@ export const SynapseNavDrawer: React.FunctionComponent<
           </a>
         </div>
         <List>
-          {currentUserProfile && (
+          {isLoggedIn && currentUserProfile && (
```

Comment:
currentUserProfile will only be true iff isLoggedIn is true (so no functional change), but I think inspecting `isLoggedIn` makes the logic more clear